packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -268,10 +291,20 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
   // quasis are the string pieces of the template literal - the parts around the interpolations.
   const literalResult = node.quasis.reduce<string>((acc, q, index): string => {
     const nodeExpression = node.expressions[index] as t.Expression | undefined;
+
+    if (
+      nodeExpression &&
+      t.isArrowFunctionExpression(nodeExpression) &&
+      t.isLogicalExpression(nodeExpression.body)
+    ) {
+      return acc + q.value.raw + ';';
+    }
+
     if (!nodeExpression) {
       return acc + q.value.raw + ';';
     }
 
+    // The following is used for CSS mixins
```

Comment:
I noticed that the following lines were used for CSS mixins. Let me know if it is used for something else or I should remove it